### ISSUE TITLE
Version 4.0.0 Erroring on Startup - Stalls server - Stop connections 

### ISSUE THREAD

**Issue by TheFlash787** [the-obsidian/DiscordBridge#63]
https://hastebin.com/itusujuvaf.java

This happens when all details are correct in the configs.

**Comment by DiamondIceNS** [comment]
Can you give me more specific information about your server JAR type and version, plus any mods/plugins you are using alongside DiscordBridge?

**Comment by TheFlash787** [comment]
Hey yes, Sorry about that:

https://files.modrealms.net/screenshots/17.11.28-16.06.18.png
https://files.modrealms.net/screenshots/17.11.28-16.06.18.png

(Above contain all of the plugins)
DiscordBridge Version: 4.0.0 (Found under Releases)
SpongeForge Version: spongeforge-1.12.2-2705-7.1.0-BETA-3168.jar

There were no Sponge errors, Only JDA errors.

Thanks!

**Comment by TheFlash787** [comment]
https://hastebin.com/guxorazome.java

This is the config file

**Comment by DiamondIceNS** [comment]
Alright, I can't immediately deduce the problem; I'll take a closer look at this late this evening (CST).

**Comment by TheFlash787** [comment]
Hey! Did you manage to find anything?

Sorry to bother you,
Thanks

**Comment by DiamondIceNS** [comment]
I can't seem to reproduce the problem.  :/

There's clearly some sort of conflicting dependency with the JSON library required by JDA, but I can't cause the problem to arise so I am not sure specifically how to fix it. I'll try specifying a JSON version in the gradle file.

**Comment by DiamondIceNS** [comment]
Could you show me where to download your versions of ModRealms and Market? I can't find them. They may be involved.

**Comment by TheFlash787** [comment]
Hello! 

Market is a plugin that was designed by the KookyKraft network (which does not use JDA), ModRealms is a plugin that is no longer running on our servers (I've redesigned it), that is pretty much just textbased with Sponge and also does not use the JDA libraries.

**Comment by DiamondIceNS** [comment]
What's likely occurring is another mod is loading before DiscordBridge with a different version of JSON than the one DiscordBidge expects. Could I ask you to try setting up a server with only Forge, Spongeforge, and DiscordBridge? If that works, add mods one by one until the problem returns and let me know which one breaks it.

Also, are you using DiscordBridge Forge or DiscordBridge Sponge? I've never encountered a Spongeforge server before and I assume you could be using either one.

**Comment by TheFlash787** [comment]
I've tried running it with a fresh install and it works fine (Using Spongeforge version). When even adding all of the plugins it works fine too. Doesnt seem to work when it has all the forge mods. The modpack is FTB Revelation 2.0.0

**Comment by DiamondIceNS** [comment]
There is no "spongeforge" version of DiscordBridge. There is a Sponge version and there is a Forge version. Which did you download from the Releases page?

**Comment by TheFlash787** [comment]
Yes Sorry, SpongeForge is Sponge. (Therefore I used the Sponge version of the mod). Forge is just forge without plugins. SpongeForge/Vanilla/API all run on the same thing, So Sponge is the correct version.

**Comment by TheFlash787** [comment]
Bump

**Comment by DiamondIceNS** [comment]
Hi, sorry for the wait, I was away this weekend.

I finally got that modpack downloaded and ran it with the version of DiscordBridge you specified, and could not reproduce the problem. But just in case, I've tried relocating the JSON library to a different name on the classpath. Let me know if this version of the plugin works for you.

https://github.com/the-obsidian/DiscordBridge/releases/tag/v4.0.1-dev
